runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkJobInvoker.java

```diff
@@ -23,7 +23,6 @@
 import javax.annotation.Nullable;
 import org.apache.beam.model.pipeline.v1.RunnerApi;
 import org.apache.beam.runners.core.construction.PipelineOptionsTranslation;
-import org.apache.beam.runners.flink.translation.utils.Workarounds;
```

Comment:
Removing the workaround and adding the new test should be in two discrete commits.

Reply:
That is debatable. I've split it up, but I do think these changes can exist together. It helps to reason why the test was introduced.